src/main/java/fi/hsl/transitdata/omm/OmmAlertHandler.java

```diff
@@ -141,6 +141,10 @@ private static long toUtcEpochMs(LocalDateTime localTimestamp, String zoneId) {
             builder.setEffect(bulletin.impact.toGtfsEffect());
             builder.setDescriptionText(bulletin.descriptions);
             builder.setHeaderText(bulletin.headers);
+            final Optional<Alert.SeverityLevel> maybeSeverityLevel = bulletin.priority.toGtfsSeverityLevel();
+            if (maybeSeverityLevel.isPresent()) {
```

Comment:
this would be the "functional way":

```   
maybeSeverityLevel.map(level -> builder.setSeverityLevel(level)) 
```   
or you might be even able to use 
```   
maybeSeverityLevel.map(BuilderTypeWhateverItIs::setSeverityLevel) 
```